receiver/chronyreceiver/metadata.yaml

```diff
@@ -74,3 +74,7 @@ metrics:
       value_type: double
     attributes:
     - leap.status
+
+tests:
+  config:
+  skip_lifecycle: true
```

Comment:
Why is this `true`?

Reply:
If you try to start the receiver, it fails looking for the chrony socket file.

Reply:
The generated tests test two scenarios:
* lifecycle, meaning we start and stop the receiver with a configuration, and make sure it is able to run. This is skipped.
* shutdown: we create the receiver and call .Shutdown on it. We check the receiver doesn't panic on this unexpected call.

Reply:
I know. so the shutdown part is failing without functional changes?

Reply:
Indeed, the current behavior of the chrony receiver is to create a chrony client before creating the receiver. This client immediately looks up a file on disk and fails. Moving the chrony client creation to start helps.

Reply:
Thanks for clarifying 👍